paymentmethod.go

```diff
@@ -0,0 +1,183 @@
+package stripe
+
+import "encoding/json"
+
+// PaymentMethodType is the list of allowed values for the payment method type.
+type PaymentMethodType string
+
+// List of values that PaymentMethodType can take.
+const (
+	PaymentMethodTypeCard        PaymentMethodType = "card"
+	PaymentMethodTypeCardPresent PaymentMethodType = "card_present"
+)
+
+// PaymentMethodCardBrand is the list of allowed values for the brand property on a
+// Card PaymentMethod.
+type PaymentMethodCardBrand string
+
+// List of values that PaymentMethodCardBrand can take.
+const (
+	PaymentMethodCardBrandAmex       PaymentMethodCardBrand = "amex"
+	PaymentMethodCardBrandDiners     PaymentMethodCardBrand = "diners"
+	PaymentMethodCardBrandDiscover   PaymentMethodCardBrand = "discover"
+	PaymentMethodCardBrandJCB        PaymentMethodCardBrand = "jcb"
+	PaymentMethodCardBrandMastercard PaymentMethodCardBrand = "mastercard"
+	PaymentMethodCardBrandUnionpay   PaymentMethodCardBrand = "unionpay"
+	PaymentMethodCardBrandVisa       PaymentMethodCardBrand = "visa"
+	PaymentMethodCardBrandUnknown    PaymentMethodCardBrand = "unknown"
```

Comment:
Minor alphabetization problem here: unknown should be above visa.

Reply:
Nice catch, fixed